tests/integration/update_cluster/minimal_hetzner/data/aws_s3_object_minimal.example.com-addons-hcloud-cloud-controller.addons.k8s.io-k8s-1.22_content

```diff
@@ -22,7 +22,7 @@ metadata:
     addon.kops.k8s.io/name: hcloud-cloud-controller.addons.k8s.io
     app.kubernetes.io/managed-by: kops
     k8s-addon: hcloud-cloud-controller.addons.k8s.io
-  name: cloud-controller-manager
+  name: hcloud-cloud-controller-manager
```

Comment:
will kops prune the old service account?

Reply:
I remember enabling pruning. Though it's not much of a problem if those stay behind.